pkg/api/apis/installplan/v1alpha1/types.go

```diff
@@ -153,12 +153,13 @@ type Step struct {
 // StepResource represents the status of a resource to be tracked by an
 // InstallPlan.
 type StepResource struct {
-	CatalogSource string `json:"sourceName"`
-	Group         string `json:"group"`
-	Version       string `json:"version"`
-	Kind          string `json:"kind"`
-	Name          string `json:"name"`
-	Manifest      string `json:"manifest,omitempty"`
+	CatalogSource          string `json:"sourceName"`
+	CatalogSourceNamespace string `json:"sourceNamespace"`
```

Comment:
Can we change `status.catalogSources` to be a list with both the `name` and `namespace` of the `CatalogSources` as well?

Reply:
`InstallPlanStatus.CatalogSources` as a `[]registry.SourceKey`?

Reply:
Done.